### PR TITLE
converted basculechecks metrics to use touchstone

### PR DESCRIPTION
**Pull request by kristinapathak**
fixes https://github.com/xmidt-org/bascule/issues/93

### CODE REVIEW

File: basculechecks/metricvalidator.go [hunk 1]
```diff
@@ -177,3 +195,23 @@ func determineEndpointMetric(endpoints []*regexp.Regexp, urlHit string) string {
 	}
 	return "not_recognized"
 }
+
+func ProvideMetricValidator(server string) fx.Option {
+	return fx.Provide(
+		fx.Annotated{
+			Name: fmt.Sprintf("%s_bascule_capability_measures", server),
+			// TODO: this will be fixed when Metric Validator gets its own New()
```
Comment (joe94) [inline basculechecks/metricvalidator.go:203]: Will this be part of another PR?

Comment (kristinapathak) [reply basculechecks/metricvalidator.go:203]: yeah: https://github.com/xmidt-org/bascule/issues/87 👍

**Review by joe94 — APPROVED** [review]
